Dev-9/Parser.java

```diff
@@ -0,0 +1,88 @@
+import java.util.*;
+
+/**
+ * Class to parse data from .ini-file on sections, on keys and keys values.
+ */
+class Parser {
+  private int first = 0;
+  private ArrayList<String> list = new ArrayList<String>();
+  private ArrayList<String> keyListWithValues = new ArrayList<String>();
+  private ArrayList<String> keyList = new ArrayList<String>();
+  private String paramValue = new String();
+
+/**
+ * @param text contain data from .ini-file.
+ * @return the list of sections.
+ */
+  public ArrayList<String> getListOfSections(String text) {
+    for(int i = 1; i < text.length(); i++) {
+      if(text.charAt(i) == '[') {
+        list.add(text.substring(first, i));
+        first = i;
+      }
+      if(i == text.length() - 1) {
+        list.add(text.substring(first, text.length()));
+        break;
+      }
+    }
+
+    return list;
+  }
+
+/**
+ * @param text contain name of section.
+ * @return the list of keys with values.
+ * This method only in class.
+ */
+  private ArrayList<String> getListOfKeysWithValues(String text, ArrayList<String> sectionList) {
```

Comment:
No need to pass "sectionList" as parameter, sectionList is equal to field "list" already stored in the class object